typer/rich_utils.py

```diff
@@ -628,13 +639,20 @@ def rich_format_help(
                 )
                 panel_to_commands[panel_name].append(command)
 
+        # Identify the longest command name in all panels
+        max_cmd_len = 0
+        for _, commands in panel_to_commands.items():
+            for command in commands:
+                max_cmd_len = max(max_cmd_len, len(command.name or ""))
```

Comment:
Depending on Tiangolo's preference, we could also rewrite this with a list comprehension, which could be slightly more efficient in doing the `max` calculation:
```suggestion
        max_cmd_len = max([len(c.name) for _, commands in panel_to_commands.items() for c in commands if c.name], default=0)
```

Reply:
Good idea! I just added that. :rocket: